botorch/test_functions/multi_fidelity.py

```diff
@@ -126,7 +131,11 @@ class AugmentedRosenbrock(SyntheticTestFunction):
     _optimal_value = 0.0
 
     def __init__(
-        self, dim=3, noise_std: float | None = None, negate: bool = False
+        self,
+        dim=3,
+        noise_std: float | None = None,
+        negate: bool = False,
+        dtype: torch.dtype = torch.double,
```

Comment:
missing dtype arg in docstring below

Reply:
Adding argument to docstring, hence resolving this conversation.